src/components/tile/tile.spec.tsx

```diff
@@ -185,3 +187,16 @@ describe("Tile", () => {
     rootTagTest(wrapper.find(StyledTile), "tile", "foo", "bar");
   });
 });
+
+test.each<TileProps["highlightVariant"]>(TILE_HIGHLIGHT_VARIANTS)(
```

Comment:
**praise:** At least this is one test the person who picks up the Tile RTL refactor won't have to worry about.